app/views/admin/partners/edit.html.erb

```diff
@@ -59,14 +60,21 @@
 
     <ul>
       <% @partner.users.each do |p_user| -%>
-        <li><%= link_to p_user.email, edit_admin_user_path(p_user) %></li>
+        <li>
+        <% if current_user.root? %>
+          <%= link_to p_user.email, edit_admin_user_path(p_user) %></li>
+        <% else %>
+          <%= p_user.email %></li>
+        <% end %>
       <% end -%>
     </ul>
 
-    <p><%= link_to 'Add new partner admin',
-                    new_admin_user_path(partner_id: @partner.id),
-                    class: "btn btn-primary mb-3 mr-2 btn-sm" %>
-    </p>
+    <% if current_user.root? %>
```

Comment:
anyone who can create a partner needs to be able to add and remove admins, so neighbourhood admins whose neighbourhood this partner resides in also need to be able to do this - probably again, who can do this should be mandated at model level on the user when given a partners id

Reply:
i don't think you can even see this page unless you are a partner admin. see this [policy](https://github.com/geeksforsocialchange/PlaceCal/blob/febc6292df99ad11339dbd1bc9204b870effc316/app/policies/partner_policy.rb#L20) logic.

might as well take the check out. unless there is a specific subset of those permissions that should only be able to "see" the users